{{cookiecutter.repo_name}}/gulpfile.js

```diff
@@ -79,8 +79,13 @@ gulp.task('extras', () => {
 gulp.task('watch', ['watchify'], () => {
   const browserSync = require('browser-sync').create()
   browserSync.init({
-    proxy: '127.0.0.1:8000',
-    files: './public/**/*'
+    proxy: '127.0.0.1:' + (parseInt(process.env.PORT, 10) - 100), // subtract 100 because foreman adds 100 to each success worker in the Procfile
```

Comment:
I'm a bit concerned about having the `- 100` offset here because if the Procfile order is changed or a new entry added, things will start to break for seemingly no reason. BUT, this is more valuable in the short-term and is something we can figure out how to fix later.